recipes/draco/meta.yaml

```
@@ -0,0 +1,42 @@
+
+{% set name = "draco" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "9f369ce924974acf9ab582fe8a9e50d86d54346460b0ac6d3808e3fcf269489c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/draco/archive/{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - draco_encoder -h
+
+about:
+  home: https://github.com/google/draco/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

I think `LICENSE` in included in the source tarball. If so, you can delete the `LICENSE.txt` file and use this instead:

```suggestion
  license_file: LICENSE
```